### DISCUSSION TITLE
Disk size

### DISCUSSION THREAD

**Discussion by linuxgokan** [abbbi/virtnbdbackup#222] [General]
Hello

Disk is 98% full. if i run a tnb backup will it fill the /tmp directory? VM is 400gb. I will get a HDD attached to the system for backup. will i have problems while taking backup?

/dev/md2       ext4   436G  405G  9.3G  98% /

**Comment by abbbi** [comment]
no, temp files are in /var/tmp/:

https://github.com/abbbi/virtnbdbackup?tab=readme-ov-file#backup-io-and-performance-scratch-files

Having 98% full root filesystem is never a good idea. No matter if you use this utility or not.
